### ISSUE TITLE
[SUGGESTION] pyarmor-webui

### ISSUE THREAD

**Issue by chwba** [dashingsoft/pyarmor#315]
`python -m pip install pywin32`
```python
import win32api
try:
	win32api.SetConsoleTitle("Pyarmor Webui")
	# Disable quick edit in CMD, as it can freeze the application
	handle = win32api.GetStdHandle(-10)
	if handle != -1 and handle is not None:
		ctypes.windll.kernel32.SetConsoleMode(handle, 128)
except:
	pass
```

Disable quickedit and insert mode in cmd programatically, so they cannot block app execution.

**Comment by jondy** [comment]
pyarmor-webui is not only for Windows, but for Linux and MacOS. This only works in Windows. If there is a common way to disable quickedit and insert mode of console, I'll add this feature.

**Comment by meanwood** [comment]
For Windows as far as I know this is the best way to do it, also for my apps, that also use and show CMD I fixed many weird problems with this, especially if the apps are being run on RDP. On Linux/macOs this problem  does not exist.

**Comment by chwba** [comment]
For Windows as far as I know this is the best way to do it, also for my apps, that also use and show CMD I fixed many weird problems with this, especially if the apps are being run on RDP. On Linux/macOs this problem does not exist

**Comment by jondy** [comment]
Add this patch, refer to https://github.com/dashingsoft/pyarmor-webui/commit/568c97271ec4de3218503d2d1027e6f94abdf87a

**Comment by chwba** [comment]
I noticed:
```
 install_requires=['pyarmor>=6.4.0'] + (
        ['win32api'] if platform == 'win32' else []),
)
```
I think you cant install win32api directly, win32 api is included in pywin32, so if you install pywin32, win32api is included.

**Comment by jondy** [comment]
Thanks, fixed it.

**Comment by jondy** [comment]
This patch has been released with pyarmor-webui 1.2.6
